.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# GitHub code owners
+# See https://help.github.com/articles/about-codeowners/
+#
+# KEEP THIS FILE SORTED. Order is important. Last match takes precedence
+
+CHANGELOG-ee.md @mstanleyjones @joaofnfernandes
```

Comment:
this is the `ce` repo, so don't think this file will be there?

Reply:
Don't know how I missed that. Got it fixed. Thanks!